src/style/App.css

```diff
@@ -12,3 +97,25 @@ li {
 .priority-urgent {
   color: red;
 }
+
+.App{
+  height: 100vh;
+}
+
+
+input[type="checkbox"],
+span,button {
+  border: none;
+  margin-right: 0.4rem;
+}
+
+span {
+  margin-right: 0.4rem;
+}
```

Comment:
Where are we putting general styles, at the top or at the beginning?

Reply:
general styles you mean when is nothing with classes? like like targeting an html tag??